pyiron/atomistics/structure/analyse.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import numpy as np
+from pyiron_base import Settings
+from sklearn.cluster import AgglomerativeClustering
+
+__author__ = "Joerg Neugebauer, Sam Waseda"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Sam Waseda"
+__email__ = "waseda@mpie.de"
+__status__ = "production"
+__date__ = "Sep 1, 2017"
+
+s = Settings()
+
+class Analyse:
+    """ Class to analyse atom structure.  """
+    def __init__(self, ref_structure):
+        """
+        Args:
+            ref_structure (pyiron.atomistics.structure.atoms.Atoms): reference Atom structure
+        """
+        self._ref_structure = ref_structure
+
+    def __repr__(self):
+        """
+        Returns: __repr__
```

Comment:
Nit: Codacy will probably complain about the 1 line docstring taking three lines.